src/TodoListItem.js

```diff
@@ -1,20 +1,21 @@
 import React from "react";
 
-//Function creates list item with remove button. 
 
-const TodoListItem = ({ todo, onRemoveTodo }) => {
+//Function creates list item with checkbox, todo.title, star, edit, remove.--sb
+
+const TodoListItem = ({ todo, onRemoveTodo, handleCheck, handleStar }) => {
   return (
       
     <li key={todo.id}>
+
+      <input value={todo.title} type="checkbox" onChange={handleCheck} />
       {todo.title}
+      <button type="button" onClick={handleStar}>Favorite</button>
+      <button type="button" >Edit</button>
       <button type="button" onClick={() => onRemoveTodo(todo.id)}>Remove Button</button>
```

Comment:
This probably doesn't matter, because I think the plan is to replace these button labels with icons in the future, but I would call this "Remove" instead of "Remove Button" to stick with the established naming convention for these three buttons ("Favorite", "Edit", etc.)